src/Xamarin.Android.Tools.AndroidSdk/Sdks/AndroidSdkUnix.cs

```diff
@@ -221,10 +221,16 @@ void AddToList (string path)
 		//
 		private static string UnixConfigPath {
 			get {
-				var p   = AppDomain.CurrentDomain.GetData (GetUnixConfigDirOverrideName)?.ToString ();
+				var p = AppDomain.CurrentDomain.GetData (GetUnixConfigDirOverrideName)?.ToString ();
 				if (string.IsNullOrEmpty (p)) {
 					p   = Environment.GetFolderPath (Environment.SpecialFolder.ApplicationData);
```

Comment:
What value does this method return under Mono? Should we just change it to:
```suggestion
					p   = Environment.GetFolderPath (Environment.SpecialFolder.UserProfile);
```

Reply:
On Mono it's doing the correct thing, this same code works perfectly on the mono-based Installer, and see:
```
$ csi
Microsoft (R) Visual C# Interactive Compiler version 3.9.0-6.21124.20 ()
Copyright (C) Microsoft Corporation. All rights reserved.

Type "#help" for more information.
> Environment.GetFolderPath (Environment.SpecialFolder.ApplicationData);
> var p = Environment.GetFolderPath (Environment.SpecialFolder.ApplicationData);
> Console.WriteLine(p);
/Users/rodrigo/.config
```
that's why I added this code as a fallback, to not break non-net6 scenarios.